src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -220,7 +223,12 @@ message CloseSharedMemoryResourcesResponse {
   map<string, bool> close_map_results = 1;
 }
 
-// Host tells the worker to load a Function
+// Host tells the worker to load a list of Functions
```

Comment:
I want to make sure we follow the right proto update practice here, ideally the [proto PR goes in first ](https://github.com/Azure/azure-functions-language-worker-protobuf/pull/63), then we have a PR in the host to just bump the version. I'd like to keep the proto changes separate from the code changes in case we need to revert

Reply:
Agreed.

Discussed offline about errors I am getting while following documentation due to unrelated histories. We will try again to bump the version after files are same.